spec/system/candidate_interface/candidate_viewing_application_spec.rb

```diff
@@ -7,6 +7,16 @@
     given_i_am_signed_in
     when_i_visit_the_site
     then_i_should_see_that_i_have_made_no_choices
+
+    #while I have not submitted an application
+    when_i_visit_the_review_page
+    then_i_should_see_that_i_have_made_no_choices
+
+    when_i_visit_the_complete_page
+    then_i_should_see_that_i_have_made_no_choices
+
+    when_i_visit_the_submit_success_page
+    then_i_should_see_that_i_have_made_no_choices
```

Comment:
Perhaps create an alias for `then_i_should_see_that_i_have_made_no_choices` so it reads more natural:

```rb
when_i_visit_the_review_page
then_i_am_on_the_application_form_page
```

Reply:
I fixed this in bd7cb3f